client/src/components/ui/itemrow/InviteRow.svelte

```diff
@@ -20,12 +21,18 @@
         email: email,
         office: office,
```

Comment:
Although this wasn't part of your diff, I'd like to address the shorthand here.
```suggestion
        email,
        office,
```